presto-main/src/main/java/com/facebook/presto/operator/aggregation/PrecisionRecallAggregation.java

```diff
@@ -113,27 +113,30 @@ public static void combine(
 
     protected static class BucketResult
     {
-        public final double totalTrueWeight;
-        public final double totalFalseWeight;
-        public final double runningTrueWeight;
-        public final double runningFalseWeight;
-        public final double left;
-        public final double right;
+        public final double threshold;
```

Comment:
Can we create getters for these variables? We don't usually make variables public unless they are static.